docs/source/guides/quickstart.md

```diff
@@ -1,48 +1,149 @@
 # NuCypher Quickstart
 
-## Ursula
 
-### Install NuCypher
+## A Note about Side-Channels
 
+The NuCypher network does not store or handle an application's data; instead - it manages access *to* application data
+Management of encrypted secrets and public keys tends to be highly domain-specific. In other words, the surrounding architecture will vary greatly depending on the throughput, sensitivity and sharing cadence of the data in question, to name but a few influencing factors.
+In all cases however, NuCypher must be integrated with a storage and transport layer in order to function properly.
+Along with the transport of ciphertexts, the application also needs to include 
+facilities for several other pieces of cryptographic material, specifically, channels
+for Alice and Bob to discover each other's public keys, as well as a way to provide the policy's public key 
+to Bob and Enrico.
+ 
+##### Side Channel Data:
+  - Secrets:
+    - Message Kits (Ciphertext)
+    - Labels
+  - Identities:
+    - Alice Verifying Key
+    - Bob Encrypting Key
+    - Bob Verifying Key
+  - Policies:
+    - Policy Encrypting Key 
+
+
+## Connecting to the NuCypher Network
+
+### Running an Ethereum Node
+
+Operation of NuCypher cryptological characters [`Alice`, `Bob`, `Ursula`] requires
+a running Ethereum node and wallet (we recommend Geth).
+
+To run a Goerli connected Geth node:
 ```bash
-$ pip3 install -U nucypher
+$ geth --goerl
```

Comment:
Can mention `--syncmode light` option. It's not 100% stable on goerli, but it's a life savior when one uses a mobile connection (or shitty hackathon internet if it is such)

Reply:
Hmm, looks like `--syncmode light` doesn't work because client's response it too slow (more than our 5 s)

Reply:
:heavy_check_mark: